package/yast2-iscsi-lio-server.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Jan 31 14:16:48 UTC 2022 - Steffen Winterfeldt <snwint@suse.com>
+
+- enhance documentation
```

Comment:
Remember, we cannot bump version without a reference. I mean, we can, but bots will complain.